base_user_role/tests/test_user_role.py

```diff
@@ -177,7 +177,7 @@ def test_default_user_roles(self):
             {"name": "USER TEST (DEFAULT ROLES)", "login": "user_test_default_roles"}
         )
         roles = self.role_model.browse([self.role1_id.id, self.role2_id.id])
-        self.assertEqual(user.role_ids, roles)
+        self.assertEqual(user.with_context(active_test=False).role_ids, roles)
```

Comment:
this won't work, you can better revert your changes to the tests and add this context to https://github.com/OCA/server-backend/blob/3f07f31b76b2f8fa89d8e84498e2bb2d44964738/base_user_role/models/user.py#L27

Reply:
Okey, I have applied the changes now and I'm waiting to see if the tests are working.